db/migrate/20170330161442_change_institution_defaults.rb

```diff
@@ -0,0 +1,48 @@
+class ChangeInstitutionDefaults < ActiveRecord::Migration
+  def change
+    change_table :institutions do |t|
+      t.change :flight, :boolean, default: nil
+      t.change :correspondence, :boolean, default: nil
+      t.change :gibill, :integer, default: nil
+      t.change :yr, :boolean, default: nil
```

Comment:
What effect does removing the default for :yr, :student_veteran, and other fields have on the `facets` object returned by institution search API? Do we now wind up with counts for true, false, and null instead of just true and false?